### PR TITLE
Added missing quotemark converter

### LINKED ISSUE #182: Missing converter: quotemark
### 💥 Missing Converter

-   `tslint-to-eslint-config` version: 1.2.0
-   `typescript-eslint` version: 2.3.2

#### TSLint Rule Name

`quotemark`

### PR DESCRIPTION
**Pull request by mauricetmeyer**
## PR Checklist

-   [x] Addresses an existing issue: fixes #182
-   [x] That issue was marked as [`status: accepting prs`](https://github.com/typescript-eslint/tslint-to-eslint-config/labels/status%3A%20accepting%20prs)

## Overview

Adds a missing rule converter for the `quotemark` rule.

### CODE REVIEW

File: src/rules/converters/quotemark.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { RuleConverter } from "../converter";
+
+export const convertQuotemark: RuleConverter = tslintRule => {
+    const notices: string[] = [];
+    const ruleArguments: any[] = [];
```
Comment (JoshuaKGoldberg) [inline src/rules/converters/quotemark.ts]: `any` is kind of a dangerous bad kid in TypeScript land. It removes a lot of type safety from its operations. Let's switch to the more specific `string[]`:

```suggestion
    const ruleArguments: string[] = [];
```

File: src/rules/converters/quotemark.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+    ["jsx-single", "jsx-double", "avoid-template"].forEach(rule => {
```
Comment (JoshuaKGoldberg) [inline src/rules/converters/quotemark.ts]: > `rule`

This confused me for a few seconds: really we're looping over the _options_ for the rule, not rules themselves. Could you please rename these `rule` variables to `option`?

File: src/rules/converters/tests/quotemark.test.ts [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+                {
+                    ruleName: "@typescript-eslint/quotes",
+                    ruleArguments: [],
+                    notices: [],
+                },
+            ],
+        });
+    });
+
+    test("conversion with an arguments", () => {
```
Comment (JoshuaKGoldberg) [inline src/rules/converters/tests/quotemark.test.ts]: 😉 

```suggestion
    test("conversion with arguments", () => {
```

File: src/rules/converters/quotemark.ts (hunk 2 above)
Comment (mauricetmeyer) [reply src/rules/converters/quotemark.ts]: Done

File: src/rules/converters/quotemark.ts (hunk 1 above)
Comment (mauricetmeyer) [reply src/rules/converters/quotemark.ts]: I can't use string[] as avoidEscape rule is an object, but I created the type below that is now used as the array type.

```ts
type QuotemarkRule = string | { avoidEscape: true }
const ruleArguments: QuotemarkRule[] = [];
```

**Pull request merged**